plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -1124,7 +1271,7 @@ function VocabBuilder:init()
 end
 
 function VocabBuilder:addToMainMenu(menu_items)
-    menu_items.vocabulary_builder = {
+    menu_items.vocabbuilder = {
         text = _("Vocabulary builder"),
         keep_menu_open = true,
```

Comment:
It feels this `keep_menu_open = true` is not really needed - I'm mostly annoyed seeing the menu still open once I'm done playing. The dict lookup history doesn't keep it open, as most other full screen widget/apps I think.

Reply:
Removed it.